libs/libdaemon/src/BaseDaemon.cpp

```diff
@@ -723,13 +723,28 @@ void BaseDaemon::initialize(Application& self)
 		std::string core_path = config().getString("core_path", "");
 		if (core_path.empty())
 			core_path = getDefaultCorePath();
-		Poco::File(core_path).createDirectories();
+
+		try
+		{
+			Poco::File(core_path).createDirectories();
+		}
+		catch (const std::exception & e)
+		{
+			logger().warning("Cannot create core_path directory [ " +  core_path + " ]" + " what(): " + e.what());
+		}
```

Comment:
Why catching just std::exception? Please note, that what() of std::exception will contain only exception name without descriptive message.
Double whitespace before core_path.
Why you embed path in square brackets and why there is padding with whitespace inside?